src/app/app.routes.ts

```diff
@@ -1,3 +1,16 @@
-import { Routes } from '@angular/router';
+import { RouterModule, Routes } from '@angular/router';
+import { MenuItems } from './menu-items/menu-items';
+import { Home } from './home/home';
+import { NgModule } from '@angular/core';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+  { path: '', redirectTo: 'home', pathMatch: 'full' },
+  { path: 'home', component: Home },
+  { path: 'menu-items', component: MenuItems },
+];
+
+@NgModule({
```

Comment:
it's older approach but for now it's fine, once after TODO implementation finished kindly revisit this implementation